src/services/ScrapboxService.ts

```diff
@@ -0,0 +1,32 @@
+import {
+  ScrapboxNotice,
+  ScrapboxNoticeAttachment,
+} from '../@types/ScrapboxNotice';
+
+const ScrapboxService = () => {
+  const isNotifiable = (attachment: ScrapboxNoticeAttachment): boolean => {
+    const regexList = [/^日報[\s　].*/, /^下書き[\s　].*/];
```

Comment:
[imo]
マッチすると通知しないやつらのマッチするやつが 0 件というのがやや分かりづらいので、どういう正規表現か書くと良さそう

```suggestion
    const ignoreTitlePatterns = [/^日報[\s　].*/, /^下書き[\s　].*/];
```

Reply:
@karur4n 
何故か commit suggestion ボタンから取り込めなかったので直接修正した
https://github.com/unabara-org/scrapbox-slack-notification/pull/1/commits/49598c0c9bd417b24a7bb5fa26296617b3fcb6bf